src/patternfly/components/Alert/alert.scss

```diff
@@ -3,215 +3,174 @@
   --pf-c-alert--BoxShadow: var(--pf-global--BoxShadow--lg);
   --pf-c-alert--BackgroundColor: var(--pf-global--BackgroundColor--100);
   --pf-c-alert--grid-template-columns: max-content 1fr max-content;
-  --pf-c-alert--grid-template-rows: 1fr auto;
+  --pf-c-alert--grid-template-rows: var(--pf-global--spacer--lg) auto auto;
```

Comment:
I don't think you need to specify rows. Giving it a fixed height creates issues if you change the size of anything in that row. Looks like maybe it's there to keep the close button from making that row so tall? The way the alert currently does it works pretty well.

<img width="646" alt="Screen Shot 2020-04-03 at 6 28 56 PM" src="https://user-images.githubusercontent.com/35148959/78412613-910b2080-75d9-11ea-908d-75d7f35aef4a.png">